models/experimental/bert/fused_ops/layernorm.py

```diff
@@ -165,17 +162,17 @@ def layernorm_2d_(x):
         W = x.get_legacy_shape()[3]
 
         # first compute the mean (m)
-        redW = tensor.reduce(x, RSUM, RW, 1.0 / W)  # -> NCH1
-        mean = tensor.reduce(redW, RSUM, RH, 1.0)  # -> NC11 (HW reduce doesn't behave well with small scaler)
+        redW = ttnn.sum(x, 3, scalar=1.0 / W)  # -> NCH1
+        mean = ttnn.sum(redW, 2)  # -> NC11 (HW reduce doesn't behave well with small scaler)
```

Comment:
These reduced by height previously. In some cases we removed Dim.W, here we basically remove Dim.H. 
Is it intentional?

Reply:
This should be functionally equivalent, first line was reduce RW which now maps to 3 (dim) and the sencond line was RH which now maps to "2"